.gitignore

```diff
@@ -53,3 +53,6 @@ export.json
 
 # TypeScript
 typechain/
+
+# Contract artifacts
+solidity/random-beacon/deployments
```

Comment:
We should declare it in the Random Beacon's file: https://github.com/keep-network/keep-core/blob/main/solidity/random-beacon/.gitignore